index.js

```diff
@@ -90,16 +111,32 @@ var User = function(server, options, next) {
               }
               // Renew expire time for each request.
               result.expire = moment().add(1, "day").format();
+              if (options.authInMemory) {
+                _.some(inMemoryTokens, function(t){
```

Comment:
@mdamt Those token data will be checked and updated (expiry) in the next cycle. Since hash is one way encryption, it wouldn't work.